drivers/bridge/setup_ip_tables.go

```diff
@@ -29,17 +70,18 @@ func (n *bridgeNetwork) setupIPTables(config *networkConfiguration, i *bridgeInt
 		return fmt.Errorf("Failed to Setup IP tables: %s", err.Error())
 	}
 
-	_, err = iptables.NewChain(DockerChain, config.BridgeName, iptables.Nat, hairpinMode)
+	_, filterChain, err := n.getDriverChains()
 	if err != nil {
-		return fmt.Errorf("Failed to create NAT chain: %s", err.Error())
+		return fmt.Errorf("Failed to setup IP tables, cannot acquire chain info %s", err.Error())
 	}
+	filterChain.Bridge = config.BridgeName
```

Comment:
If a chain is a property of the driver and is shared by all the networks which use the bridge driver, then it doesn't seem right to store the `BridgeName` in the `filterChain` especially since `n.getDriverChains` returns a pointer to the chain